includes/AMP/Sanitization.php

```diff
@@ -449,7 +449,7 @@ protected function get_sanitizers(): array {
 				'publisher'         => $story->get_publisher_name(),
 				'poster_images'     => array_filter( $poster_images ),
 				'video_cache'       => $video_cache_enabled,
-				'semantic_headings' => $this->experiments->is_experiment_enabled( 'semanticHeadingTags' ),
+				'semantic_headings' => true,
```

Comment:
We can also remove all usage of `semantic_headings` and update the two sanitizers accordingly.